build.gradle

```diff
@@ -21,14 +22,14 @@ repositories {
 dependencies {
   implementation gradleApi()
   implementation localGroovy()
+  compile 'com.android.tools.build:gradle:3.6.1'
 
   compileOnly 'com.google.code.findbugs:jsr305:3.0.2'
 
   testImplementation(group: 'org.spockframework', name: 'spock-core', version: '1.1-groovy-2.4') {
     exclude group: 'org.codehaus.groovy', module: 'groovy-all'
   }
-  testImplementation 'com.android.tools.build:gradle:3.3.1'
-  testImplementation 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.21'
+  testImplementation 'org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.61'
```

Comment:
Just a few callouts since this PR is aging just a bit (thanks for your awesome efforts!). Unless you want to target 1.3.61 specifically we're already to 1.3.72 stable version. Same with Gradle on line #25, we're at 3.6.3.

Reply:
Shouldn't affect if it gets merged as-is IMO, just if there's time/reason before @arturdm gets their eyes on it.

Reply:
yeah moving targets are hard. I'll go through and refresh this PR one more time. Personally I finally gave up waiting and did a custom internal plugin to handle the job this repo was serving.

Reply:
How custom? I'd love to see only if you're able to share.

Reply:
it's just a derivative version of this repo. ~200 lines of kotlin to replace this whole project. 

custom in the sense: not just a copy/paste from here.